### PR TITLE
Update mathtype from 7.4.2 to 7.4.3

### PR DESCRIPTION
**Pull request by ran-dall**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/mathtype.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'mathtype' do
-  version '7.4.2'
-  sha256 'd2b2bfcd37f212edd1d7241d11ecc345cafe6adffcc2fc7a0e1565afb2759e89'
+  version '7.4.3'
```
Comment (vitorgalvao) [inline Casks/mathtype.rb]: Why not `7.11.0`?

Comment (ran-dall) [reply Casks/mathtype.rb]: Cause I think it verified it with the output request.

Comment (ran-dall) [reply Casks/mathtype.rb]: I'll fix it shortly.

Comment (suschizu) [reply Casks/mathtype.rb]: hey @ran-dall - you were absolutely right with version 7.4.3 (#69554)

Comment (ran-dall) [reply Casks/mathtype.rb]: @suschizu  It's the ML, but it's good to know that it's working correctly. Every day getting closer to a public release. ⭐️

**Pull request closed without merging**